test/automated/ansible/roles/install-python/tasks/main.yaml

```diff
@@ -1,4 +1,4 @@
 ---
-include_tasks: dnf.yaml
-include_tasks: brew.yaml
+- include_tasks: dnf.yaml
+- include_tasks: brew.yaml
 ...
```

Comment:
please add a new line here

Reply:
Maybe this could be a good reason to unify IDE's settings ;-) 

<img width="384" alt="Screenshot 2022-05-03 at 10 53 42" src="https://user-images.githubusercontent.com/3380451/166427403-0a1b656a-a3ea-4d67-8814-7741483cd6d2.png">

Reply:
done